src/test_driver/mbed/unit_tests/main/main.cpp

```diff
@@ -63,8 +62,14 @@ int main()
 
     ChipLogProgress(NotSpecified, "Mbed unit-tests application run");
 
-    int status = RunRegisteredUnitTests();
-    ChipLogProgress(NotSpecified, "CHIP test status: %d", status);
+    // Migration to pw_unit_tests was not done for mbed since the unit-testing FW used to run out of flash and was disabled from
```

Comment:
Is there an github issues to track to re-enable this after we split tests to multiple smaller binaries?

Reply:
Can you ping me the number on slack too? Then I can add it to the project tracker.

Reply:
The issue is that MBED has no maintainer anymore for a long time now, so we have no platform support for issues. 

We could work on it, however without a maintainer for a platform, us taking over running and configuring tests on a platform seeems rough. Unsure if it should be on an official project tracker as a result.

Reply:
Issue link should probably also be added to https://github.com/project-chip/connectedhomeip/blob/master/.github/workflows/examples-mbed.yaml which is where this started (we disabled mbed tests because of flash)

Reply:
Issue created and added to the comments (of the yaml as well)